omnibus-base-goerli.toml

```diff
@@ -5,15 +5,18 @@ include = [
     "tomls/settings-testnet.toml",
     "tomls/core.toml",
     "tomls/permissions.toml",
-    #"tomls/collaterals/snx.toml",
     "tomls/collaterals/weth.toml",
```

Comment:
`collaterals/weth` includes `oracles/eth.toml`. This is the chainlink oracle which would mean we're not using pyth and ERC7412 for base-goerli like we do in the competition preset.. I dont have context what we actually want, but thought it was worth mentioning